sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -269,11 +277,9 @@ def visit_transform(self, transform_node):
               pcoll.element_type, transform_node.full_label)
           key_type, value_type = pcoll.element_type.tuple_types
           if transform_node.outputs:
-            from apache_beam.runners.portability.fn_api_runner.translations \
-              import only_element
             key = (
-                None if None in transform_node.outputs.keys() else only_element(
-                    transform_node.outputs.keys()))
+                None if None in transform_node.outputs.keys() else
+                DataflowRunner._only_element(transform_node.outputs.keys()))
```

Comment:
The conditional is a tautology; just return  `_only_element(transform_node.outputs.keys())`.